### PR TITLE
Updates to testing project

### PR DESCRIPTION
**Pull request by joseph-robertson**
Repackage `project_resstock_testing.zip` on S3 to include AMY 2012 (leap year) and AMY 2014 weather files. Update the testing project's location-related tsv files to sample these AMY files.

### CODE REVIEW

File: project_testing/housing_characteristics/Geometry Building Type FPL.tsv [hunk 1]
```diff
@@ -1,2 +1,2 @@
 Option=Mobile Home	Option=Single-Family Detached	Option=Single-Family Attached	Option=2 Unit	Option=3 or 4 Unit	Option=5 to 9 Unit	Option=10 to 19 Unit	Option=20 to 49 Unit	Option=50 or more Unit
-0.111111111	0.111111111	0.111111111	0.111111111	0.111111111	0.111111111	0.111111111	0.111111111	0.111111111
+0.166666667	0.166666667	0.333333333	0.055555556	0.055555556	0.055555556	0.055555556	0.055555556	0.055555556
```
Comment (shorowit) [inline project_testing/housing_characteristics/Geometry Building Type FPL.tsv]: What's the reason for this change?

Comment (joseph-robertson) [reply project_testing/housing_characteristics/Geometry Building Type FPL.tsv]: I was intending to sample one third SFD, one third SFA, one third MF. But with the fractions before, it was oversampling MF.

Comment (shorowit) [reply project_testing/housing_characteristics/Geometry Building Type FPL.tsv]: Got it. 👍
